catalog/files/impl/src/main/java/org/projectnessie/catalog/files/adls/AdlsFileSystemOptions.java

```diff
@@ -47,16 +44,16 @@ public interface AdlsFileSystemOptions {
   /** Configure the retry strategy. */
   Optional<AdlsRetryStrategy> retryPolicy();
 
-  /** Mandatory, if any {@link AdlsRetryStrategy} is configured. */
+  /** Mandatory, if any {@code retry-policy} is configured. */
```

Comment:
These changes are probably for the generated docs right? Could we use `{@link AdlsRetryStrategy retry-policy}`?

Reply:
Nope. The reason is a javadoc-headache - resolving a link is a ton of work - you'd have to implement all the resolving logic on your own - given the complexity of that I didn't do that in the doclet.